.NET/Microsoft.Recognizers.Definitions.Common/Japanese/DateTimeDefinitions.cs

```diff
@@ -25,70 +25,72 @@ public static class DateTimeDefinitions
+      public const string YearNumRegex = @"(?<year>((1[5-9]|20)[0-9]{2})|2100)年?";
+      public const string SimpleYearRegex = @"(?<year>([0-9]{2,4}))(\s*年)?";
+      public const string ZeroToNineIntegerRegexCJK = @"[一二三四五六七八九十零壹贰叁肆伍陆柒捌玖〇两千俩倆仨０１２３４５６７８９]";
+      public const string DynastyStartYear = @"元";
+      public const string RegionTitleRegex = @"(昭和|平成|令和|大正|明治|寛政|享和|文化|文政|天保|弘化|嘉永|安政|万延|文久|元治|慶応)";
+      public static readonly string DynastyYearRegex = $@"(?<dynasty>{RegionTitleRegex})(?<biasYear>({DynastyStartYear}|\d{{1,2}}|({ZeroToNineIntegerRegexCJK}){{1,3}}))年?";
+      public static readonly string DateYearInCJKRegex = $@"(?<yearCJK>({ZeroToNineIntegerRegexCJK}{{2,4}}|{DynastyYearRegex}))年?";
       public const string WeekDayRegex = @"(?<weekday>日曜日?|月曜日?|火曜日?|水曜日?|木曜日?|金曜日?|土曜日?)\s?";
       public const string LunarRegex = @"(农历|初一|正月|大年|旧暦)";
-      public static readonly string DateThisRegex = $@"(这个|这一个|这|这一|本|今週|そ)(的|の)?({WeekDayRegex}|日)";
+      public static readonly string DateThisRegex = $@"(这个|这一个|这|这一|本|今週|そ|こ)(的|の)?({WeekDayRegex}|日)";
       public static readonly string DateLastRegex = $@"(上一个|上个|上一|上|最后一个|最后|先週|最後)(的|の)?({WeekDayRegex}|日)";
-      public static readonly string DateNextRegex = $@"(下一个|下个|下一|下|来週)(的|の)?{WeekDayRegex}";
+      public static readonly string DateNextRegex = $@"(下一个|下个|下一|下|来週|次)(的|の)?{WeekDayRegex}";
       public const string WoMLastRegex = @"^\b$";
       public const string WoMPreviousRegex = @"^\b$";
       public const string WoMNextRegex = @"^\b$";
       public const string SpecialMonthRegex = @"(先月|来月|今月|前月|再来月|昨月|先々月|ぜんげつ|(せん)?せんげつ|さくげつ|らいげつ|こんげつ)";
       public const string SpecialYearRegex = @"(昨年|再?来年|今年|前年|ことし|さ?らいねん|きょねん|さくねん|去年)";
-      public const string SpecialDayRegex = @"((いっ)?さくじつ|おとつい|最近|前天|后天|昨天|明天|今天|今日?|明日|一?昨?昨日|一昨日|大后天|大前天|後天|大後天|きょう|あす|あした|きのう|明々後日|(弥)?明後日|この日|前日|二日前|おととい|し?あさって)((から|の)([\d１２３４５６７８９十一二三四五六七八九]*)(日|週)間?(先|後|前)?(の(日曜日?|月曜日?|火曜日?|水曜日?|木曜日?|金曜日?|土曜日?))?)?";
-      public const string SpecialDayWithNumRegex = @"^[.]";
-      public static readonly string WeekDayOfMonthRegex = $@"((({MonthRegex}|{MonthNumRegex}|((这个|这一个|这|这一|本|今|上个|上一个|上|上一|去|下个|下一个|下|下一|明)月))(的|の)\s*)(?<cardinal>(第[一二三四五]个?)|(第[12345１２３４５])|最后一个?)\s*{WeekDayRegex})";
-      public const string ThisPrefixRegex = @"这个|这一个|这|这一|本|今";
+      public const string SpecialDayRegex = @"((いっ)?さくじつ|おとつい|最近|前天|后天|明日から二日|昨日の2日前|昨日から4日|今日から二日|今日から4日|昨日から2日間|昨天|明天|今天|今日|明日|一?昨?昨日|一昨日|大后天|大前天|後天|大後天|きょう|あす|あした|きのう|明々後日|(弥)?明後日|この日|前日|二日前|おととい|し?あさって|私の一日|この間|次の日|その日|最後の日)";
+      public const string SpecialDayWithNumRegex = @"((いっ)?さくじつ|おとつい|最近|前天|后天|昨天|明天|今天|今日?|明日|一?昨?昨日|一昨日|大后天|大前天|後天|大後天|きょう|あす|あした|きのう|明々後日|(弥)?明後日|この日|前日|二日前|おととい|し?あさって|私の一日|この間|次の日|その日)(から|の)?([\d１２３４５６７８９十一二三四五六七八九]*|数)(日|月|週|個)間?(先|後|前|以内)?(の(?<weekday>日曜日?|月曜日?|火曜日?|水曜日?|木曜日?|金曜日?|土曜日?))?";

```

Comment:
Same here and in the next regex.

Reply:
Please check all regexes for the same case.

Reply:
done